### PR TITLE
replace the ci-trigger action with a external one

### PR DESCRIPTION
**Pull request by keewis**
I think this helps reducing the maintenance burden (I also use this elsewhere). However, it would probably be good to move the action to a different organization (`xarray-contrib`? `pydata`?).

- [x] Passes `pre-commit run --all-files`

cc @andersy005

### CODE REVIEW

**Review by andersy005 — APPROVED** [review]
> However, it would probably be good to move the action to a different organization (xarray-contrib? pydata?)


👍🏽 I'm in favor of hosting this under either `xarray-contrib` or `pydata` (whichever one is the easiest ...).

**Comment by keewis** [comment]
`xarray-contrib`, then: I would be able to do the migration myself.

File: .github/workflows/ci-additional.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 2
-      - uses: ./.github/actions/detect-ci-trigger
+      - uses: keewis/ci-trigger@v1
```
Comment (andersy005) [inline .github/workflows/ci-additional.yaml:22]: I'm assuming you will update this one too...

Comment (keewis) [reply .github/workflows/ci-additional.yaml:22]: thanks, I missed that one

**Pull request merged**